src/client/src/apps/MainRoute/routes/ShellRoute.tsx

```diff
@@ -85,6 +94,45 @@ const ShellRoute: React.FC<Props> = ({ header }) => {
     </StatusBar>
   );
 
+  //@ts-ignore
+  if (window.fin && window.fin.me.isWindow) {
+    const OfTabTheme = createGlobalStyle<{ theme: Theme }>`
+      .lm_tabs {
```

Comment:
This can be extracted out for easier reuse. In addition to specific classes that we can override the styles of, Openfin provides css variables as well.